login_tests/resource.robot

```diff
@@ -0,0 +1,59 @@
+*** Settings ***
+Documentation     A resource file with reusable keywords and variables.
+...
+...               The system specific keywords created here form our own
+...               domain specific language. They utilize keywords provided
+...               by the imported SeleniumLibrary.
+Library           SeleniumLibrary
+Library           Dialogs
+
+*** Variables ***
+${SERVER}         login-dxt360.sonarplatform.com
+${BROWSER}        Chrome
+${DELAY}          0
+${VALID USER}     demo.analytics@dxt360.app
+${VALID PASSWORD}    p3iN%u49
```

Comment:
ini harus pakai environment variable ya @gibrandarmawan

Reply:
bisa via terminal mas

Reply:
install dotenv, then add file .env and put all sensitive parameters in it, make .env file ignored by git

then add also .env.example file to give what are variable the used by application so other devs will know it

Reply:
intinya sensitive values should not shared in git.

Reply:
oke mas, saya ganti dulu